superset-frontend/src/SqlLab/components/SaveDatasetModal/index.tsx

```diff
@@ -218,7 +220,9 @@ export const SaveDatasetModal = ({
         ...formDataWithDefaults,
         datasource: `${datasetToOverwrite.datasetid}__table`,
         ...(defaultVizType === 'table' && {
-          all_columns: datasource?.columns?.map(column => column.column_name),
+          all_columns: datasource?.columns?.map(
+            column => column.name || column.column_name,
```

Comment:
I've asked this before, but I don't remember the answer. Why can't we standardize this? This is definitely going to cause bugs in the future, having two possible options for the attribute name.

Can't we normalize the schema early, if they have to be stored differently, instead of normalizing it late here?

Reply:
looking into this now @betodealmeida, i think it was just a difference between how sqllab consumes columns vs. explore